tdrs-backend/tdpservice/email/templates/data-submitted.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
```

Comment:
This file was used for testing purposes and was left as an example. We may want to format this differently, I just copied the html a link in the epic somewhere.